ord_schema/visualization/reaction.html

```diff
@@ -797,10 +850,10 @@ <h4>Measurements</h4>
                         <td>{{ measurement|oneof('value')|product_measurement_value }}</td>
```

Comment:
This line (L850) is giving me an error when I try rendering first reaction from my Biginelli dataset:

[biginelli.pb.zip](https://github.com/open-reaction-database/ord-schema/files/6369636/biginelli.pb.zip)

`TypeError: getattr(): attribute name must be string`

This happens because `message.WhichOneof(name)` is `None` for `name == 'value'` in the `_oneof` filter function. i.e., there are `IDENTITY` measurements without a value defined

Reply:
Thanks; fixed.